CIP-1694/README.md

```diff
@@ -0,0 +1,1142 @@
+
+* **DReps**<br/>
+  The DRep vote threshold that must be met as a percentage of *active voting stake*.<br/>
+  A value of - means that DReps votes do not apply.
+
+* **SPOs**<br/>
+  The SPO vote threshold which must be met as a percentage of the stake held by all stake pools.<br/>
+  A value of - means that SPO votes do not apply.
+
+| Governance action type                                         | CC   | DReps    | SPOs     |
+| :---                                                           | :--- | :---     | :---     |
+| 1. Motion of no-confidence                                     | \-   | $P_1$    | $Q_1$    |
+| 2<sub>a</sub>. New committee/quorum (_normal state_)           | ✓    | $P_{2a}$ | \-       |
+| 2<sub>b</sub>. New committee/quorum (_state of no-confidence_) | \-   | $P_{2b}$ | $Q_{2b}$ |
+| 3. Update to the Constitution                                  | ✓    | $P_3$    | \-       |
+| 4. Hard-fork initiation                                        | ✓    | $P_4$    | $Q_4$    |
+| 5<sub>a</sub>. Protocol parameter changes, network group       | ✓    | $P_{5a}$ | \-       |
+| 5<sub>b</sub>. Protocol parameter changes, economic group      | ✓    | $P_{5b}$ | \-       |
+| 5<sub>c</sub>. Protocol parameter changes, technical group     | ✓    | $P_{5c}$ | \-       |
+| 5<sub>d</sub>. Protocol parameter changes, governance group    | ✓    | $P_{5d}$ | \-       |
+| 6. Treasury withdrawal                                         | ✓    | $P_6$    | \-       |
+| 7. Info                                                        | ✓    | $P_7$    | \-       |
+
+> **Warning**
+> For treasury withdrawals, the withdrawal threshold is the **total** amount of Lovelace that is withdrawn by the action, not the amount of any single withdrawal if the action specifies more than one withdrawal.
+
+Each of these thresholds is a governance parameter.
+The initial thresholds should be chosen by the Cardano community as a whole.
+
+> **Note**
+> It may make sense for some or all thresholds to be adaptive with respect to the Lovelace that is actively registered to vote.
+> For example, a threshold could vary between 51% for a high level of registration and 75% for a low level registration.
+> Moreover, the treasury threshold could also be adaptive, depending on the total Lovelace that is being withdrawn,
+> or different thresholds could be set for different levels of withdrawal.
+
+> **Note**
+> To achieve legitimacy, the minimum acceptable threshold should be no less than 50% of the delegated stake.
+
+
+##### Restrictions
+
+Apart from _Treasury withdrawals_ and _Infos_, we include a mechanism for ensuring that governance
+actions of the same type do not accidentally clash with eath other in an unexpected way.
+
+Each governance action must include the previous governance action ID of its given type.
+This means that two actions of the same type can be enacted at the same time,
+but they must be *deliberately* designed to do so.
```

Comment:
…, but hard forks and parameter updates are different types and a hard fork could remove (or change the semantics) of parameters in a parameter update. Shouldn't parameter updates then also have a reference to the hard fork they depend on?

Reply:
It's a solid point, one that @nicarq brought up recently as well. I like the idea of coming to consensus on the genesis files by way of the info action: https://github.com/JaredCorduan/CIPs/blob/voltaire-v1/CIP-1694/README.md#the-versatility-of-the-info-governance-action